src/WorkflowCore/Services/BackgroundTasks/WorkflowConsumer.cs

```diff
@@ -100,10 +100,6 @@ await _persistenceStore.ScheduleCommand(new ScheduledCommand()
 
         private async Task SubscribeEvent(EventSubscription subscription, IPersistenceProvider persistenceStore, CancellationToken cancellationToken)
```

Comment:
The name of this method no longer makes sense in the context of these changes... perhaps `TryProcessSubscription`?

Reply:
@danielgerlag done.